Celeste.Mod.mm/Patches/Commands.cs

```diff
@@ -241,5 +241,35 @@ private class MeshData {
         private static float Float(string data) {
             return float.Parse(data, CultureInfo.InvariantCulture);
         }
+
+        [Command("mainmenu", "go to the main menu")]
+        private static void CmdMainMenu() {
+            Engine.Scene = new OverworldLoader(Overworld.StartMode.MainMenu);
```

Comment:
Maybe add these before `Engine.Scene` to be consistent with vanilla `CmdOverworld()`? Although I don't know what's the use of it either *:theo:*

```suggestion
            if (SaveData.Instance == null) {
                SaveData.InitializeDebugMode(true);
                ((patch_SaveData) SaveData.Instance).CurrentSession_Safe = new Session(AreaKey.Default);
            }
            Engine.Scene = new OverworldLoader(Overworld.StartMode.MainMenu);
```

Reply:
I don't think it's necessary. I was referring to vanilla `CmdTitlescreen()`, which is just one line of code